### PR TITLE
[CI] Enable 201911 branch PR check

### PR DESCRIPTION
**Pull request by liushilongbuaa**
#### Why I did it
Enable azure pipeline PR check for 201911 branch.
#### How I did it
Add 201911 branch to trigger. But Test stage is failing now, so I commented the Test stage leaving a TODO. When Test stage passed, uncomment it.
#### How to verify it
In this PR, check is already enabled.
#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -17,6 +17,7 @@ pr:
     include:
       - master
       - 202012
+      - 201911
```
Comment (xumia) [inline azure-pipelines.yml:20]: @lguohan , we need to add 201811 as well, right?

**Comment by xumia** [comment]
@lguohan , do we need to add 201811 as well? Need to add a new commit in 201811 branch.

**Review by liat-grozovik — APPROVED** [review]

**Comment by liat-grozovik** [comment]
@liushilongbuaa can you please check why it is not working?

File: azure-pipelines.yml [hunk 2]
```diff
@@ -45,84 +46,83 @@ stages:
       - name: vs
       - name: broadcom
       - name: mellanox
-
-- stage: Test
```
Comment (xumia) [inline azure-pipelines.yml:49]: If you want to disable the Test stage, please use.
condition: false

@qiluo-msft , do you think we can only enable the build without Test? It should be better than without any check in 2001911.

**Review by xumia — APPROVED** [review]

**Review by qiluo-msft — APPROVED** [review]

File: azure-pipelines.yml (hunk 2 above)
Comment (qiluo-msft) [reply azure-pipelines.yml:49]: Agree. At lease we have some build checker, better than nothing.

**Pull request merged**